lib/components/SInputImage.vue

```diff
@@ -71,6 +71,8 @@ function onFileSelect(e: Event) {
 }
 
 function onFileDelete() {
+  fileInput.value!.value = ''
+
   emit('update:model-value', null)
   emit('change', null)
```

Comment:
probably we don't need these 👀  changing value should trigger onFileSelect IG

Reply:
ah, no, somehow that isn't happening. weird. keeping them should be fine IG.

Reply:
👍